src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
```

Comment:
Unused import statement 'java.util.Locale'; if it's not used elsewhere, it should be removed to keep the code clean.